lib/features/offline_mode/work_manager/sending_email_worker.dart

```diff
@@ -76,9 +76,11 @@ class SendingEmailWorker extends Worker {
       HiveCacheConfig().setUp()
     ]);
 
-    SendingQueueInteractorBindings().dependencies();
-    SendEmailInteractorBindings().dependencies();
-    SendingQueueBindings().dependencies();
+    await Future.sync(() {
```

Comment:
srr, but why it can fix the issue?

Reply:
SendingQueueController uses MailboxDashboardController so we need to bind it.